docs/how_to/create_toggles_report_for_devstack_or_sandbox.rst

```diff
@@ -25,8 +25,7 @@ First create this empty directory. Now populate it with data based on either of
 When creating report to compare multiple environments
 +++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-For each env, create a directory named following this name pattern: `<env_name>_env`.
-Populate these env directories with their state value files (json).
+For each env, create a directory named following this name pattern: `<env_name>_env`. Populate these env directories with their state value files (json).
```

Comment:
This, and other sections, feel like they belong elsewhere:
1. This doc claims to be "How-to: Enabling toggles report for a new IDA", which should be a list of steps needed to accomplish this.  Maybe the doc I want is to edx.org specific?
2. The "Problems with current approach" feel like they belong in code comments or maybe an ADR about some implementation, etc.  Maybe small parts of it belong in this file as a warning?
3. I don't know where this section belongs?  This seems like more manual instructions on how to gather data for generating the report.  This seems important, like for devstack, sandboxes, Open edX, etc., but this feels like the wrong how-to.

Reply:
Agreed. I really struggled with how to refactor this doc, and the others -- the information is spread out across about 4 repositories and some combination of duplicated, out of date, and not properly stratified into Open edX/edx.org. I'll put something on your schedule.

Reply:
Sounds about right.